test/images/scripts/continuous-atomic.setup

```diff
@@ -16,11 +16,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
-set -e
+set -ex
 
 # Disable device renaming for predictable names between different qemus
 grubby --update-kernel=ALL --args="net.ifnames=0 biosdevname=0"
 
+# HACK: docker fails without /etc/resolv.conf
+# https://bugzilla.redhat.com/show_bug.cgi?id=1448331
+mkdir -p /etc/systemd/system/docker.service.d
+printf "[Service]\nExecStartPre=/bin/sh -c 'echo -n >> /etc/resolv.conf'\n" > /etc/systemd/system/docker.service.d/resolv.conf
+systemctl daemon-reload
```

Comment:
If you do this in the images in the first place, there is no need to do this in the systemd unit -- the image could just run `touch /etc/resolv.conf` and be done with it? But as I said above, I'm not convinced that this is a good idea, as this would then apply to all other (non-docker) tests too and potentially hide other issues.